.github/workflows/test.yml

```diff
@@ -28,4 +28,4 @@ jobs:
         run: |
           cargo build
           cargo test
-          cargo clippy -- -Dwarnings
+          cargo clippy -- -Dwarnings --allow clippy::blocks_in_conditions
```

Comment:
Is it possible to disable this inline instead of globally?

Reply:
Unfortunately, since the lint is being triggered by a macro (hence "inside" the macro), there is no way of disabling it inline.
But it can be disabled per file instead of globally. Please see commit [a02f64e](https://github.com/wbcsd/pathfinder-use-case-001/pull/20/commits/a02f64e9c87fe42aeafc3e739cacb4fb2f01d780)
Thank you!